src/main/scala/com/cognite/spark/datasource/DataPointsRelation.scala

```diff
@@ -258,8 +258,8 @@ class DataPointsRelation(
     }
 
     Tuple2(
-      Try(timestampLimits.filter(_.isInstanceOf[Min]).min).toOption.map(_.value),
-      Try(timestampLimits.filter(_.isInstanceOf[Max]).max).toOption.map(_.value))
+      Try(timestampLimits.filter(_.isInstanceOf[Min]).max).toOption.map(_.value),
```

Comment:
Shouldn't we have the minimum of the minimums, and maximum of the maximums?

Reply:
If users want "select * from _cdp.datapoints where name=<something> and timestamp > 1 and timestamp > 2 ", we should give them timestamp > 2. So that's the max of min.

Reply:
Consider `select * from _cdp.datapoints where name=some_name and timestamp > 1 or timestamp > 2`. It's better to return too much data, than to return too little data.

Reply:
Hmm... Currently in getTimestampLimit(filters), OR is ignored, only AND is parsed. And adding support for OR will need more work than using min of min and max of max, since we can have "timestamp > num1 or timestamp < num2"

Reply:
That's a good point. Still, I don't see any harm in using the safer choice of the minimum min, even if it ends up returning a bit of extra data in your example. Spark will always do its own filtering afterwards.

If you really want to leave this in I'm ok with it as well, but then it should be commented and a note should be added to the commented out OR parsing, warning that this needs to be changed when/if we add support for OR.